osaca/parser/directive.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+from osaca.parser.operand import Operand
+
+
+class DirectiveOperand(Operand):
+    def __init__(self, name=None, parameter_id=None, comment_id=None):
+        super().__init__(name)
+        self._parameter_id = parameter_id
+        self._comment_id = comment_id
```

Comment:
Why does the directive have a comment attribute? I believe this shouldn't be in here.

Reply:
https://github.com/RRZE-HPC/OSACA/commit/7ad3438af543c9d6991cac6fed9450ab6adb1500